policyengine_us/parameters/gov/usda/snap/income/deductions/utility/limited/allowance.yaml

```diff
@@ -1,184 +1,235 @@
-description: A household is eligible for the Limited Utility Allowance if it does not have separate heating and cooling costs, but does have at least two other utilities. Also known as the 'basic' utility allowance.
+description: A household is eligible for the Limited Utility Allowance if it does
+  not have separate heating and cooling costs, but does have at least two other utilities.
+  Also known as the 'basic' utility allowance.
 AK:
   2010-01-01: 0
   2021-10-01: 0
+  2023-10-01: 0.0
 AL:
   2010-01-01: 0
   2021-10-01: 389
+  2023-10-01: 434.0
```

Comment:
```suggestion
  2023-10-01: 434
```

Reply:
Just some formatting standards

Reply:
OK. I'll format it as int (vs. float). To your previous point, this brings up the larger conversation about how PolicyEngine stores and uses parameter values. It may be time to move toward a more complex database or parameters.

Reply:
Thank you @eqberkovich, more than happy to get together and discuss options in the near future

Reply:
Pavel -- How attached are you to the int vs. float format? It turns out some of the values are floats (there's an 83.96).
The units are currency-USD, so maybe should format as XXX.XX ?

Reply:
Let's continue without the decimal for consistency with the rest of the repo, and discuss a potential switch in https://github.com/PolicyEngine/policyengine-us/discussions/4663

Reply:
OK. I'll round the floats to ints.